### PR TITLE
Import Export functions

### LINKED ISSUE #878: Explain people how do not lose wallet
It's not clear how to backup and restore wallet (and do not lose your NTZ)

Related to #872

### LINKED ISSUE #879: Improve import/export/logout dialogs
Related #872 

Currently, it's pure functionality without any explanations

### LINKED ISSUE #882: Remove DBButton styled component
Doesn't look like it is used anywhere:
/components/Dashboard/styles.js - DBButton

### LINKED ISSUE #883: As a user I want to be able to request more play chips
As a new customer, I want to be able to request more play chips when I have lost all chips on the account (NTZ and ETH). 

There should be a limitation for such requests - once per hour.

### PR DESCRIPTION
**Pull request by VonIobro**
- [x] Resolves #879
- [x] Resolves #883 
- [x] Resolves #882 
- [x] Resolves #878

Issues: 
- [x] Modals require two clicks to close, modal is created twice on button click: fixed bug
- [x] Logout button should only be used in an extraodinary situation, so it should be placed in the 'Advanced' tab as a seperate function

### CODE REVIEW

**Comment by VonIobro** [comment]
@sunify ready to merge.

Possible new issues: 
1. when reseting the account, the modal for 'wallet_loaded' is vague and should have explanation, also this should maybe not even pop up, it could just be a CONFIRM_DIALOG to let user know a new account/wallet has been created
2. After creating a new account, no error is thrown in console anymore, however it seems a browser refresh is required to get the new account/wallet loaded
  a. click reset dialog button, from Dashboard > Advanced > Account Reset > Reset Account
  b. click the greyed-out avatar image (that would normally lead to dashboard), because there is no account information, router redirects to log-in page

**Comment by sunify** [comment]
@VonIobro we do not create a new account on logout. An account will be created only on next visit.
About logout dialog. We should give people ability to restore account after logout.

File: app/components/Header/index.js [hunk 1]
```diff
@@ -1,94 +1,87 @@
...
+
+                  <Balances>
+                    {babzBalance && <span>{formatNtz(babzBalance)} NTZ</span>}
+                    {weiBalance && <span>{formatEth(weiBalance, 2)} ETH</span>}
+                  </Balances>
+                </StyledUserName>
+              </StyledUser>
+            }
+            location={location}
+            menu={[]}
```
Comment (sunify) [inline app/components/Header/index.js]: Need to remove this prop

Comment (VonIobro) [reply app/components/Header/index.js]: commented out in e4c48d4

**Pull request merged**